packages/design-system/src/components/Autocomplete/Autocomplete.tsx

```diff
@@ -27,6 +27,9 @@ import uniqueId from 'lodash.uniqueid';
 export interface AutocompleteItems {
   id?: string;
   name?: string;
+  children?: React.ReactNode;
+  className?: string;
+  isResult?: boolean;
```

Comment:
[question] is there any sort of documentation that is needed for the `AutocompleteItems` type? Specifically the `isResult` prop? Asking because would another dev know how to use this property for correct accessibility?

Reply:
d013bde